tests/ui/setup.cfg

```diff
@@ -2,5 +2,5 @@
 addopts = -r=a -vs --showlocals --tb=short
 sensitive_url = mozilla\.(com|org)
 xfail_strict = true
-DJANGO_SETTINGS_MODULE = settings_test
-base_url = http://olympia.dev:8000
+DJANGO_SETTINGS_MODULE = settings
+base_url = http://olympia.dev:80
```

Comment:
Oh gosh, this might have solved most of our problems eh?

Reply:
It was!!

Reply:
😍